README.md

```diff
@@ -6,6 +6,10 @@ Uses [elasticsearch official image](https://hub.docker.com/_/elasticsearch)
 
 `ddev get ddev/ddev-elasticsearch`
 
+### Using Elasticsearch 8
+
+After executing `ddev get`, navigate to the `.ddev/docker-compose.elasticsearch.yaml` file and modify the image version on line 6 to the desired Elasticsearch 8 version. Additionally, uncomment line 25 to utilize a specific Elasticsearch 8 configuration.
```

Comment:
It will be better for them to create a `.ddev/docker-compose-elasticsearch-version.yaml` with just the needed change. That way they can upgrade this add-on, etc.

Reply:
It did not work changing the `image:` using `docker-composer.elasticsearch-whatever.yaml` so I choosed the override file.

Updated the readme accordingly. Please have a look.

Reply:
See https://github.com/ddev/ddev-elasticsearch/pull/16#discussion_r1378497470